doc/source/cd/methods/chisquaredrift.ipynb

```diff
@@ -112,7 +112,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3",
```

Comment:
Usual comment wrt to notebooks; ideally would reset these changes to reduce file diffs.